crate/server/src/core/implementation.rs

```diff
@@ -453,6 +454,13 @@ impl KMS {
                     ),
                 }
             }
+            CryptographicAlgorithm::RSA => {
```

Comment:
I'm kinda off with that. Asymmetric keypairs have been generated in `crate/utils` for now and doing it for RSA in `crate/server` breaks this pattern.

Reply:
done in 2cb3bf5be8520d76ac94ee884cbd3a06c18763d8